CHANGELOG.md

```diff
@@ -6,14 +6,71 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/).
 
 ## Upcoming release
 
-### Added
+nothing yet
 
-* Support for Bazel 2.0.0.
+### [0.12.0] - 2020-03-DD TODO set day
 
-### Changed
+## Highlights
+
+* Various improvements to Windows support.
+
+* Support for Bazel 2.0.0
+
+* Minimum supported Bazel version is now 0.29
+
+## Added
+
+* `haskell_repl` now has a `hie_bios` output group
+* Added support for [hrepl](https://github.com/google/hrepl)
```

Comment:
The PR has some nice details, so would be good to link to it here https://github.com/tweag/rules_haskell/pull/1210

Reply:
Done in cb537c2.